.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -5,6 +5,14 @@ title: "(module name): (short issue description)"
 labels: [bug, needs-triage]
 assignees: []
 body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        **⚠️ Please read this before filling out the form below:**  
+        If the bug you are reporting is a security-related issue or a security vulnerability,
+        please report it via [Report a security vulnerability](https://github.com/aws/aws-cdk/security/advisories/new) instead of this template.
```

Comment:
Thank you @gracelu0 , not sure if this is the template we should use related to issues that might not be related to any CVE per say, but i'll let team also take a look to chime in. IMO, if its not related to a CVE maybe we have a new type of bug template with security checkbox or related attributes to be populated.

Reply:
This is the link a customer would get if they selected "Report a security vulnerability" on the `New Issue` page.

Reply:
We will review the content of the `Report a security vulnerability` link with security team to review the best way how customers should report security issues. For now, we will refer the customers to this link, till we know what should be the content of that link.